metadata-db-app/src/migrations/058_add_provider_id_to_variables_table.clj

```diff
@@ -0,0 +1,24 @@
+(ns migrations.058-add-provider-id-to-variables-table
+  "Adds `provider_id` column to variables table."
+  (:require
+   [clojure.java.jdbc :as j]
+   [config.mdb-migrate-helper :as h]))
+
+(defn up
+  "Migrates the database up from version 57 to 58."
+  []
+  (println "migrations.058-add-provider-id-to-variables-table up...")
+  ;; At this point in time, UMM-Vars support is still being developed, so we
+  ;; don't care about what's in the database right now, and the new field
+  ;; can't be null, so:
+  (h/sql "TRUNCATE TABLE METADATA_DB.cmr_variables")
+  (h/sql (str "ALTER TABLE METADATA_DB.cmr_variables "
+              "ADD provider_id VARCHAR(255) NOT NULL "
```

Comment:
That's the only place I could find 'provider' and 'max' on the same line in the code base, so I made those two constants public and required the ns in the migration. Let me know if you think these constants should be moved someplace more general ...